src/tree_schema.h

```diff
@@ -1568,69 +1606,166 @@ struct lysc_node {
     void *priv;                      /**< private arbitrary user data, not used by libyang */
 };
 
+struct lysc_node_action_inout {
+    union {
+        struct lysc_node node;       /**< implicit cast for the members compatible with ::lysc_node */
+        struct {
+            uint16_t nodetype;       /**< LYS_INPUT or LYS_OUTPUT */
+            uint16_t flags;          /**< [schema node flags](@ref snodeflags) */
+            uint8_t hash[LYS_NODE_HASH_COUNT]; /**< schema hash required for LYB printer/parser */
+            struct lys_module *module; /**< module structure */
+            struct lysc_node *parent; /**< parent node (NULL in case of top level node) */
+            struct lysc_node *next;  /**< next sibling node (NULL if there is no one) */
+            struct lysc_node *prev;  /**< pointer to the previous sibling node \note Note that this pointer is
+                                          never NULL. If there is no sibling node, pointer points to the node
+                                          itself. In case of the first node, this pointer points to the last
+                                          node in the list. */
+            const char *name;        /**< Notification name (mandatory) */
+            const char *dsc;         /**< description */
+            const char *ref;         /**< reference */
+            struct lysc_ext_instance *exts; /**< list of the extension instances ([sized array](@ref sizedarrays)) */
+            struct lysc_when **when; /**< list of pointers to when statements ([sized array](@ref sizedarrays)) */
+            void *priv;              /** private arbitrary user data, not used by libyang */
+        };
+    };
+
+    struct lysc_node *data;          /**< first child node (linked list) */
```

Comment:
Rename to `child` to unfiy with the rest of the nodes?

Reply:
yes, make sense not only in compiled nodes but also in the parsed nodes